package.json

```diff
@@ -68,6 +68,10 @@
     "@babel/preset-react": "^7.12.7",
     "@babel/runtime": "^7.12.5",
     "@codastic/react-positioning-portal": "^0.7.0",
+    "@jsonforms/core": "^3.2.1",
+    "@jsonforms/react": "^3.2.1",
+    "@jsonforms/vanilla-renderers": "^3.2.1",
+    "@storybook/jest": "^0.2.3",
```

Comment:
can you uninstall @storybook/jest and will this still work?  
I don't see it being used here.

Reply:
It breaks the whole thing. I see it's being used elsewhere.

Reply:
Ok. Thanks for investigating it.